eth/test/utils/TestWorld.ts

```diff
@@ -1,9 +1,9 @@
-import type { DarkForest } from '@darkforest_eth/contracts/typechain';
+import { DarkForest } from '@darkforest_eth/contracts/typechain';
```

Comment:
Why was type removed here?
```suggestion
import type { DarkForest } from '@darkforest_eth/contracts/typechain';
```